easybuild/easyconfigs/a/Anaconda/Anaconda-1.8.0-Linux-x86_64.eb

```diff
@@ -0,0 +1,23 @@
+import os
+
+easyblock = 'Binary'
+
+name = 'Anaconda'
+version = '1.8.0-Linux-x86_64'
+
+homepage = 'https://store.continuum.io/cshop/anaconda/'
+description = """Completely free enterprise-ready Python distribution for large-scale data processing, predictive analytics, and scientific computing"""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = ['https://repo.continuum.io/archive/']
+sources = ['%s-%s.sh' % (name, version)]
+qa = {
+            r'Please, press ENTER to continue': '\n',
+            r'Do you approve the license terms? [yes|no]': 'yes',
+            r'Or specify an different location below': '%(installdir)s/tmp/', 
+            r'Do you wish the installer to prepend the Anaconda install location to PATH': 'no',
+     }
+install_cmd = 'sh %s-%s.sh'  % (name,version)
+postinstallcmds  = ['mv %(installdir)s/tmp/* %(installdir)s/', 'rm -Rf %(installdir)s/tmp/']
```

Comment:
`tmp` subdir contains all files after installation. I can't use `'Or specify an different location below': '%(installdir)s/'` instead because Anaconda will try to `mkdir` the `%(installdir)s` and since EasyBuild already made it this will give a permission error. So instead I write to `tmp` and then copy it back afterwards.

My latest commit uses `rmdir` instead of `rm -Rf`.